os/include/tinyara/mm/mm.h

```diff
@@ -671,6 +695,9 @@ void mm_addfreechunk(FAR struct mm_heap_s *heap, FAR struct mm_freenode_s *node)
 /* Functions contained in mm_size2ndx.c.c ***********************************/
 
 int mm_size2ndx(size_t size);
+#ifdef CONFIG_MM_REALTIME_SUPPORT
```

Comment:
What about the below?
```suggestion
#ifdef CONFIG_MM_REALTIME_SUPPORT
int mm_size2ndx_realtime(FAR struct mm_heap_s *heap, size_t size);
#else
int mm_size2ndx(size_t size);
#endif
```
?

Reply:
Thanks for the comment, but it's an additional function in case of realtime support.